lib/skinny_controllers/lookup/ensure_existence.rb

```diff
@@ -87,13 +78,7 @@ def policy_lookup(qualified_name)
     end
 
     def operation_lookup(qualified_name)
-      return if qualified_name.blank?
-      klass = qualified_name.safe_constantize
-
-      # Return if the constant exists, or if we can't travel
-      # up any higher.
-      return klass if klass
-      return unless qualified_name.scan(/::/).count > 1
```

Comment:
It looks like this method is slightly different from the other two, so the extracted method isn't entirely equivalent is functionality. 

The first couple lines could def be extracted though from all 3 methods. It seems like it's just the first two methods you've identified that have a full 4 lines of shared code.